paf-mvp-core-js/src/log.ts

```diff
@@ -3,7 +3,7 @@ export class Log {
   private readonly id: string;
   private readonly color: string;
 
-  constructor(id: string, color: string) {
+  constructor(id: string, color = 'black') {
```

Comment:
I see different colors per module (if I understood well). Should we want to have a unique color for all PAF modules?

Reply:
There is no convention yet, but my personal opinion is:
- yes, on the frontend I would like PAF, PAFUI and CMP, Audit to have the same color. @jwrosewell seemed not to agree.
- on the backend, since all PAF modules will run in the same process, I reconned it was interesting to use different colors for the different "modules".